DuplicateAssemblyScanner/DuplicateAssemblyScanner/Settings.cs

```diff
@@ -0,0 +1,124 @@
+namespace DuplicateAssemblyScanner {
+    using ColossalFramework.UI;
+    using DuplicateAssemblyScanner.Structs;
+    using DuplicateAssemblyScanner.Util;
+    using ICities;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Generates the settings screen based on cached results from the assembly scanner.
+    /// </summary>
+    public class Settings {
+        // Markers used in checkbox labels
+        private const string MARKER_BLANK = " ";
+        private const string MARKER_ENABLED = "*";
+
+        // Cache of scan results
+        private static Dictionary<string, int> duplicates_;
```

Comment:
The .NET naming convention suggests either `PascalCase` or `camelCase` for static fields. The underscore should be used as a prefix for `_instanceFields`.

Reply:
Will fix as part of #19 for v1.2.1